docs/content/report-ingress-status.md

```diff
@@ -0,0 +1,11 @@
+---
+title: Reporting Status of Ingress Resources
+
+description: 
+weight: 100
+doctypes: ["concept"]
+toc: true
+---
+
+
+This doc is now available at https://docs.nginx.com/nginx-ingress-controller/configuration/global-configuration/reporting-resources-status/
```

Comment:
delete this doc and add an alias in the new file's frontmatter

Reply:
done